content/doc/book/system-administration/admin-password-reset-instructions.adoc

```diff
@@ -0,0 +1,38 @@
+---
+layout: section
+---
+
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+= Reset Jenkins administrator password
+
+As a system administrator, you can reset the Jenkins admin password.
+In case you have forgotten or lost the password, this page provides instructions on how you can reset the admin password.
+
+== Reset the administrator password
+
+1. Log in to your Jenkins controller and edit the Jenkins configuration file (`config.xml`) inside your `jenkins/` or `$JENKINS_HOME` folder.
+2. Look for the **useSecurity** option and change it from **true** to **false** manually.
+3. Save your file and close it.
+4. Restart the Jenkins service to apply changes. (This time it will not ask for a password.)
+5. On the dashboard, click on **Manage Jenkins** in the navigation pane on the left side of the page.
```

Comment:
```suggestion
5. On the dashboard, select  *Manage Jenkins* in the navigation pane on the left side of the page.
```
Instead of click, use select. This makes it so the instructions are more applicable across different devices.